MacrobenchmarkSample/functions/src/handlers.ts

```diff
@@ -0,0 +1,150 @@
+import { EventContext } from "firebase-functions";
+import * as F from "firebase-functions";
+import { TestMatrix } from "firebase-functions/lib/providers/testLab";
+import { Storage } from "@google-cloud/storage";
+import { log } from "./logger";
+import { Benchmark } from "./schema";
+import { MetricServiceClient } from "@google-cloud/monitoring";
+import { uploadMetrics } from "./monitoring";
+import * as express from 'express';
+
+// The key under which well known device configurations are saved.
+// This is an environment variable.
+const DEVICE_CONFIGURATIONS = 'DEVICE_CONFIGURATIONS';
+
+// The name of the output JSON file.
+const BENCHMARK_RESULT_FILE = 'androidx.benchmark.macrobenchmarks.test-benchmarkData.json';
```

Comment:
should package be a parameter? how do you specify custom parameters to the functions?

Reply:
Made this an environment variable.